src/public/ai-client-setup.js

```diff
@@ -0,0 +1,44 @@
+var appInsights = window.appInsights
+		|| function(a) {
```

Comment:
has this been copied from somewhere? 
this code is quite cryptic

Reply:
The code is from Microsoft, here: https://docs.microsoft.com/en-us/azure/azure-monitor/app/sharepoint

It is kind of an obfuscated code and was much uglier before I formatted it on IDE. :)